lib/src/layer/overlay_image_layer.dart

```diff
@@ -149,10 +149,9 @@ class OverlayImageLayer extends StatelessWidget {
     final map = MapCamera.of(context);
     return ClipRect(
       child: Stack(
-        children: <Widget>[
-          for (var overlayImage in overlayImages)
-            overlayImage.buildPositionedForOverlay(map),
-        ],
+        children: overlayImages
+            .map((image) => image.buildPositionedForOverlay(map))
+            .toList(growable: false),
```

Comment:
I'm not sure how well the dart compiler is optimized but for my understanding creating an empty list and adding items one by one should take slightly more resources than mapping an old list to a new one because the length of the new list is already known. But it's really questionable if this performance really has an impact.

Reply:
Do you still prefer to have these changes reverted? As you like.

Reply:
reverted in f73cb9cadc2e3dfa5ca2e1243e4b4ec41cd6c4f0